app/views/dashboard/show.html.erb

```diff
@@ -1,35 +1,42 @@
-<div class="govuk-grid-row">
+<div class="govuk-grid-row govuk-!-margin-bottom-6">
   <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-l">Hi <%= current_user.full_name %>!</h1>
-    <p class="govuk-body"> You are logged in</p>
-    <% if current_user.early_career_teacher? %>
-      <h2 class="govuk-heading-l">Your Core Induction Programme</h2>
-      <% current_user.course_years.each do |course_year| %>
-        <h2 class="govuk-heading-l"><%= course_year.title %></h2>
-        <% course_year.modules_with_progress(current_user).each do |course_module| %>
-          <li class="app-task-list__item">
-            <span>
-              <%= govuk_link_to course_module.title, year_module_path(course_module.course_year, course_module) %>
-            </span>
-            <%= render ProgressLabelComponent.new progress: course_module.progress %>
-          </li>
-        <% end %>
-        <% if course_year.course_modules.empty? %>
-          <p>No course modules were found!</p>
-        <% end %>
-      <% end %>
-      <% if current_user.course_years.empty? %>
-        <p>You have not been enrolled in Core Induction Programme yet!</p>
-      <% end %>
-    <% end %>
-    <% if current_user.mentor? %>
-      <h2 class="govuk-heading-l">Your list of mentees</h2>
-      <% current_user.mentor_profile.early_career_teachers.each do |early_career_teacher| %>
-        <p><%= early_career_teacher.full_name %></p>
-      <% end %>
-      <% if current_user.mentor_profile.early_career_teachers.empty? %>
-        <p>You have no mentees</p>
-      <% end %>
-    <% end %>
+    <div class="blue-header govuk-!-padding-left-3 govuk-!-padding-right-3 govuk-!-padding-top-6 govuk-!-padding-bottom-9 govuk-!-margin-bottom-6">
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-1 blue-header__title">Welcome Joe</h1>
+      <a href="#" class="blue-header__text">Change username</a>
+    </div>
+
+    <!--
+    <h2 class="govuk-heading-l">Modules progress overview</h2>
+
+    <div class="govuk-grid-row govuk-!-margin-bottom-6">
+      <div class="govuk-grid-column-one-third-from-desktop">
+        <div style="background-color:#F3F2F1; padding: 10px;">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">1</h1>
+          <p class="govuk-!-margin-bottom-1">Session done</p>
+        </div>
+      </div>
+      <div class="govuk-grid-column-one-third-from-desktop">
+        <div style="background-color:#F3F2F1; padding: 10px;">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">0</h1>
+          <p class="govuk-!-margin-bottom-1">Modules completed</p>
+        </div>
+      </div>
+      <div class="govuk-grid-column-one-third-from-desktop">
+        <div style="background-color:#F3F2F1; padding: 10px;">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">8</h1>
+          <p class="govuk-!-margin-bottom-1">Modules to complete</p>
+        </div>
+      </div>
+    </div>
+    -->
```

Comment:
Just kill this :)

Or, if you think we want to keep them, make them work. We can figure out the progress for modules and lessons in total, although it's a bit harder, so maybe it should be a separate ticket.